schemas/event.yml

```diff
@@ -128,6 +128,27 @@
 
         In case the two timestamps are identical, @timestamp should be used.
 
+    - name: start
+      type: date
+      description: >
+        event.start contains the date when the event started.
+
+        This timestamp is distinct from @timestamp in that @timestamp contains
+        the processed timestamp. For logs these two timestamps can be different
+        as the timestamp in the log line and when the event is read for example
+        by Filebeat are not identical. `@timestamp` must contain the timestamp
+        extracted from the log line, event.start when the log line is read.
+        The same could apply to package capturing where @timestamp contains the
+        timestamp extracted from the network package and event.created when the
+        event was created.
+
+        In case the two timestamps are identical, @timestamp should be used.
+
+    - name: end
+      type: date
+      description: >
+        event.end contains the date when the event ended.
```

Comment:
You have a nice description in the PR of this with an example. Could you add this here?